frontend/packages/integration-tests-cypress/views/navigation.ts

```diff
@@ -0,0 +1,9 @@
+export const navigation = {
```

Comment:
Thanks for doing this helper, unfortunately a similar helper was just [merged](https://github.com/openshift/console/pull/6521/files#diff-bfd2ae71744a1dd4227111900750f99fR14). Pls uses `nav` helper -thanks